package.json

```diff
@@ -30,40 +30,40 @@
   },
   "homepage": "https://github.com/tng/next-generation-scrum-poker#readme",
   "dependencies": {
-    "@playwright/test": "^1.32.1",
-    "@preact/preset-vite": "^2.4.0",
-    "@rollup/plugin-replace": "^5.0.1",
-    "@rollup/plugin-typescript": "^9.0.2",
+    "@playwright/test": "^1.32.3",
+    "@preact/preset-vite": "^2.5.0",
+    "@rollup/plugin-replace": "^5.0.2",
+    "@rollup/plugin-typescript": "^11.1.0",
     "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/preact": "^3.2.2",
-    "@types/aws-lambda": "^8.10.108",
-    "@types/node": "^16.18.3",
-    "@types/ws": "^8.5.3",
-    "@typescript-eslint/eslint-plugin": "^5.43.0",
-    "@typescript-eslint/parser": "^5.43.0",
-    "aws-sdk": "^2.1354.0",
-    "chart.js": "^4.0.1",
+    "@testing-library/preact": "^3.2.3",
+    "@types/aws-lambda": "^8.10.114",
+    "@types/node": "^16.18.25",
+    "@types/ws": "^8.5.4",
+    "@typescript-eslint/eslint-plugin": "^5.59.1",
+    "@typescript-eslint/parser": "^5.59.1",
+    "aws-sdk": "^2.1365.0",
+    "chart.js": "^4.2.1",
     "classnames": "^2.3.2",
     "dequal": "^2.0.3",
-    "eslint": "^8.27.0",
+    "eslint": "^8.39.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "html-minifier-terser": "^7.0.0",
-    "husky": "^8.0.2",
-    "jsdom": "^20.0.2",
-    "lint-staged": "^13.0.3",
-    "nodemon": "^2.0.20",
+    "html-minifier-terser": "^7.2.0",
+    "husky": "^8.0.3",
+    "jsdom": "^21.1.1",
+    "lint-staged": "^13.2.1",
+    "nodemon": "^2.0.22",
     "npm-run-all": "^4.1.5",
-    "preact": "^10.11.3",
-    "preact-render-to-string": "^5.2.6",
-    "prettier": "^2.7.1",
-    "rimraf": "^3.0.2",
-    "rollup": "^2.79.1",
+    "preact": "^10.13.2",
+    "preact-render-to-string": "^6.0.2",
+    "prettier": "^2.8.8",
+    "rimraf": "^5.0.0",
+    "rollup": "^3.21.0",
     "ts-node": "^10.9.1",
-    "typescript": "^4.8.4",
-    "vite": "^3.2.4",
-    "vitest": "^0.25.2",
-    "vitest-canvas-mock": "^0.2.1",
-    "ws": "^8.11.0"
+    "typescript": "^5.0.4",
+    "vite": "^4.3.2",
```

Comment:
I am surprised that this worked out of the box.I was running into commonJS vs ES module issues, which is why I extracted this into a separate issue.

Reply:
I just had one shot where I updated all dependencies together and also threw away node_modules and lock file first before installing. Maybe you get issues if you try a more incremental approach.

Reply:
Clearing the `node_modules`was probably a good idea that I did not try.